docs/pages/components/icon.md

```diff
@@ -634,6 +634,29 @@ This example will load the same set of icons from the jsDelivr CDN instead of yo
 </script>
 ```
 
+### SVG Sprites
+
+To improve performance you can use a SVG sprites to avoid multiple trips for each SVG.
+
+For now we only support referencing external URLs, meaning that we don't inline and cache the result. Make sure you have the right cache headers to avoid extra requests in production.
```

Comment:
I was curious about this and gave it a look on localhost. Apparently the browser is smart enough with SVG spritesheets to load it once and cache it.